src/engine/util/webxr/raycaster.ts

```diff
@@ -0,0 +1,108 @@
+import {
+  Raycaster,
+  Vector3,
+  BufferGeometry,
+  LineBasicMaterial,
+  Line,
+  Matrix4,
+} from "three";
+
+/** ThreeJS Line, extended with an Update method */
+class SCLine extends Line {
+  constructor(
+    bufferGeo: THREE.BufferGeometry,
+    material: THREE.LineBasicMaterial
+  ) {
+    super(bufferGeo, material);
+  }
+  Update(): void {}
+}
+
+/**
+ * Sandcastle's own Raycaster; WebXR-friendly syntactic sugar over threeJS's Raycaster
```

Comment:
Might as well be consistent about how we type Three.js when it's in documentation. I think the proper spelling is capitalized with .js